src/Protocol/ActivityPub/Processor.php

```diff
@@ -927,7 +927,16 @@ private static function processContent(array $activity, array $item)
 			$restrictions = [];
 		}
 
-		// @todo Store restrictions
+		$item['restrictions'] = null;
+ 		foreach ($restrictions as $restriction) {
+			if ($restriction == Tag::CAN_REPLY) {
```

Comment:
Same question about these constants, shouldn't they be named `CANT_*` ?

Reply:
No, the tags are indeed meant positively. They describe the capabilities.